src/Components/Message/Banner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="c-application c-banner" :class="[computedType]">
+		<div class="c-banner--img">
+			<slot name="img" />
+		</div>
+		<div class="c-banner--container">
+			<div class="c-banner--container--title">
```

Comment:
```suggestion
			<div class="c-banner--container-title">
```
dash를 2개 붙이는 것은 컴포넌트명과 추가 속성을 구분하기 위함이었고, container와 title에는 dash를 1개만 쓰는게 맞겠네요. 10, 13라인도 동일합니다.